CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` when updating to this version!
 
+- Breaking: Migrated IRC commands to relevant Helix endpoints. This will require re-auth using the `bot_login` endpoint in order to utilize the required functions --> Announce. (#2141)
```

Comment:
This needs a more thorough instruction for what the bot operator needs to do

Reply:
I have copied something similar from the v1.48 release. Have a gander and let me know what you think.